action.yml

```diff
@@ -10,6 +10,9 @@ inputs:
   token:
     description: Used to pull node distributions from go-versions.  Since there's a default, this is typically not supplied by the user.
     default: ${{ github.token }}
+outputs:
+  go-version:
+    description: 'The installed Go version. Useful when given a version range as input.'
 runs:
   using: 'node12'
```

Comment:
@mcdonnnj you should do a `git rebase main` - some of this is out of date.

Reply:
Sorry, with no attention on this pull request it fell off my radar.

Reply:
It happens to the best of us @mcdonnnj - just figured it might shake out any merge issues for you. 👍